tpgtools/api/clouddeploy/samples/updatecanary.delivery_pipeline.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "{{resource_name}}",
```

Comment:
```suggestion
    "name": "{{pipeline}}",
```
Should fix the current VCR error

Reply:
Thanks for the suggestion, I'll run the test couple times locally to make sure that's all the remaining problem.